src/components/GameItem.js

```diff
@@ -4,10 +4,35 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
-    height: PropTypes.number.isRequired,
-    layer: PropTypes.number.isRequired,
+  constructor(props){
+    super(props);
+    this.state = ({
+      className: 'game-item',
+      itemType: this.props.eachItem
+    });
   }
+  // propTypes = {
```

Comment:
You should include `propTypes` and add the callback function to the list of `propTypes`.

Something like this:

```javascript
static propTypes = {
  height: PropTypes.number.isRequired,
  layer: PropTypes.number.isRequired,
  onItemClickedCallback: PropTypes.func.isRequired,
}